libs/BehaviorKit/source/BehaviorKit.cpp

```diff
@@ -92,6 +92,11 @@ void BehaviorKit::working()
 	_videokit.displayImage("/fs/home/img/system/robot-face-smiling-slightly.jpg");
 }
 
+void BehaviorKit::fileExchange()
+{
+	_videokit.displayImage("/fs/home/img/system/robot-file-exchange-reception.jpg");
```

Comment:
let's start with that:

```suggestion
	_videokit.displayImage("/fs/home/img/system/robot-file_exchange.jpg");
```

I'll make something generic after the release